.claude/CLAUDE.md

```
@@ -139,63 +139,54 @@ make patches    # または bash tools/patches/apply.sh
+```
 
-### herdr / cmux の使い分け
+CLI は `/Applications/Orca.app/Contents/Resources/bin/orca`（PATH は .zshrc で追加済み）。ランタイム未起動なら `orca open` を先に実行。
 
-- `CMUX_WORKSPACE_ID` が設定されている（= cmux 内）→ **cmux 系スキルを優先**
-- cmux 外で、`herdr status server` が `status: running` を返す → **herdr 系スキルを使う**
-- SSH リモート・ヘッドレス・軽量に済ませたい → herdr（`herdr --remote` でリモートアタッチ）
+### 旧スキルからの対応
 
-### スキル対応表
+cmux-* / herdr-* スキルは 2026-09 に廃止（git 履歴から参照可）：
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)